### ISSUE TITLE
brin_summarize_new_values returns 0 when it should return more than 0

### ISSUE THREAD

**Issue by BaiShaoqi** [greenplum-db/gpdb#9054]
### Greenplum version or build
master branch commit 32cc07cc77a07be30da0ea9d2a6a2d3a498339f9

### Expected behavior

`brin_summarize_new_values` should return more than 0 when insert data after create the brin index.
like postgres 9.6 shows the 
```
select brin_summarize_new_values('brin_index');
 brin_summarize_new_values
---------------------------
                       691
(1 row)
```

### Actual behavior
```
select brin_summarize_new_values('brin_index');
 brin_summarize_new_values
---------------------------
                         0
(1 row)
```

### Step to reproduce the behavior
```
create table t1 as select generate_series(1,100000000) as c1;
create index brin_index on t1 using brin(c1);
insert into t1  SELECT generate_series(100000000,120000000) AS c1;
```

**Comment by d** [comment]
Thanks for reporting! This is an interesting bug to look into

**Comment by asimrp** [comment]
Is it because the function `brin_summarize_new_values()` does not dispatch the operation to QEs?  An index on master, like the base relation, would always be empty.

**Comment by lij55** [comment]
tested on master and it is fixed now (comit 71664c2a).

```sql
test=# select brin_summarize_new_values('brin_index');
 brin_summarize_new_values
---------------------------
                       172
```
